plugin/pry.vim

```diff
@@ -16,6 +16,8 @@ function! pry#insert()
     let text = get(g:pry_map, &filetype)
     call feedkeys('o', 'i')
     call feedkeys(text)
+    call feedkeys("\<Esc>")
+    call feedkeys(":w\<CR>")
```

Comment:
I like the `<Esc>` but I don't think writing is automatically is a good practice. Can we remove that?